### ISSUE TITLE
Not an issue, just a question

### ISSUE THREAD

**Issue by ekivolowitz** [balisujohn/straw#1]
TLDR; Create UI to screen grab an application, send that png to tesseract to parse, save the resulting text file and write that into output.txt?

**Comment by balisujohn** [comment]
Exactly! straw supports both output to file and output to stdout for dynamic use-cases . I will add usage instructions as well as flags for output formats soon. When you run straw, it prompts you to select a subset of the screen, and it subsequently will record the characters in that subset of the screen. For example, you could set up straw to monitor the newest message line of a text chat or the estimated time to completion field of a application wizard. Planned features include more intelligent character recognition, and multi area recording.  The fundamental idea is to be able to create an unholy interface with badly designed or intentionally non-compliant software.
